src/sql/workbench/contrib/notebook/browser/outputs/gridOutput.component.ts

```diff
@@ -143,7 +165,7 @@ class DataResourceTable extends GridTableBase<any> {
 			this.instantiationService.createInstance(SaveResultAction, SaveResultAction.SAVEEXCEL_ID, SaveResultAction.SAVEEXCEL_LABEL, SaveResultAction.SAVEEXCEL_ICON, SaveFormat.EXCEL),
 			this.instantiationService.createInstance(SaveResultAction, SaveResultAction.SAVEJSON_ID, SaveResultAction.SAVEJSON_LABEL, SaveResultAction.SAVEJSON_ICON, SaveFormat.JSON),
 			this.instantiationService.createInstance(SaveResultAction, SaveResultAction.SAVEXML_ID, SaveResultAction.SAVEXML_LABEL, SaveResultAction.SAVEXML_ICON, SaveFormat.XML),
-			this.instantiationService.createInstance(NotebookChartAction, this)
+			this.instantiationService.createInstance(NotebookChartAction, this, this.cellOutput.metadata.azdata_chartOptions !== undefined)
```

Comment:
You're already passing in this - couldn't this logic just live inside the NotebookChartAction? Not a huge deal here but trying to figure out what a bool parameter maps to annoys me